src/views/pages/index.jsx

```diff
@@ -1,24 +1,21 @@
 import React from 'react';
-import { Link } from 'react-router';
 import { getAllModels } from '../../lib/api-client';
-import { getModelsForNewItem } from '../../lib/api-client';
+import { getAllItems } from '../../lib/api-client';
 export default class Index extends React.Component {
 
     allModels(e) {
         getAllModels();
     }
 
-    newItem(e) {
-      getModelsForNewItem();
+    allItems(){
```

Comment:
Because it's hard to do that? Could this all be done for the all models thing too then?